langserver/completion.go

```diff
@@ -452,3 +480,20 @@ func getEditRange(location *cache.Location, oldname string) (editRange protocol.
 
 	return
 }
+
+// getMatches returns matches for an empty string
+func getMatches(names []string) fuzzy.Matches {
+	var (
+		match   fuzzy.Match
+		matches fuzzy.Matches
+	)
+
+	match.MatchedIndexes = append(match.MatchedIndexes, 0) // match the first rune for all
+	match.Score = 0                                        // keep same score for all
+	for i, name := range names {
+		match.Str = name
+		match.Index = i
+		matches = append(matches, match)
```

Comment:
I think we want to score alphabetically here.

Reply:
okay